operator-common/src/main/java/io/strimzi/operator/common/operator/resource/AbstractNamespacedResourceOperator.java

```diff
@@ -262,6 +297,23 @@ protected T patchOrReplace(String namespace, String name, T desired)   {
         return operation().inNamespace(namespace).withName(name).patch(PatchContext.of(PatchType.JSON), desired);
     }
 
+    protected T patchOnly(Reconciliation reconciliation, String namespace, String name, T desired) {
```

Comment:
What is the logic behind naming it `patchOnly`? I'm not sure I understand what would be the alternative to it.